From ef9c4d9a8d919430188191f0265d6732d7009914 Mon Sep 17 00:00:00 2001
From: Xufeng Zhang <xufeng.zhang@windriver.com>
Date: Wed, 13 Jul 2011 15:45:34 +0800
Subject: [PATCH 11/12] p1022ds: enable SEC channel remapping for SMP case

Interrupt handling workload can be distributed into
different cores if data is submitted by different cores.
This will enhance IPSEC performance in certain setups.

Extracted from vendor drop QorIQ-NONDPAA-SDK-V1-20110429_ltib.iso
linux-2.6.35-qoriq-p1022ds-enable-SEC-channel-remapping-fo.patch.

Signed-off-by: Dave Liu <daveliu@freescale.com>
Integrated-by: Xufeng Zhang <xufeng.zhang@windriver.com>
---
 arch/powerpc/boot/dts/p1022ds.dts |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/powerpc/boot/dts/p1022ds.dts b/arch/powerpc/boot/dts/p1022ds.dts
index 603cc11..748796f 100644
--- a/arch/powerpc/boot/dts/p1022ds.dts
+++ b/arch/powerpc/boot/dts/p1022ds.dts
@@ -554,6 +554,8 @@
 			fsl,channel-fifo-len = <24>;
 			fsl,exec-units-mask = <0xbfe>;
 			fsl,descriptor-types-mask = <0x3ab0ebf>;
+			fsl,multi-host-mode = "dual";
+			fsl,channel-remap = <0x3>;
 		};
 
 		sata@18000 {
-- 
1.7.0.4

